src/main/resources/solr/setup/core-collection.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env sh
+
+if [ $# -eq 0 ]; then
+  echo "usage: sh core-collection.sh [COLLECTION_NAME]"
+  exit 0
+fi
+
+# Set specific field types and multiValued field properties in Solr schema
+curl -X POST -H 'Content-type:application/json' --data-binary '{
```

Comment:
One suggestion might be to put the binary data in a file, and the rest of the curl command directly inline in the docs. The advantage of this is that a script is completely opaque, where you can see, oh, it's a curl command with a bunch of json data... I'll look in the file if I really want to learn about all the data...

Same with ACL anthology docs.

Reply:
Will clean this up in separate PR